learn/getting_started/algolia_migration.md

```diff
@@ -0,0 +1,249 @@
+# Migrating from Algolia to Meilisearch
+
+This guide will walk you through exporting your Algolia data and indexing it in Meilisearch using a Node.js script. To skip directly to the finished script, [click here](#finished-script).
+
+This guide also includes comparisons of Meilisearch and Algolia's [APIs](#apis), [index settings, parameters](#settings-and-parameters), and [support for the `instantsearch.js` library](#front-end-components).
+
+This is a practical guide for current Algolia users making the transition to Meilisearch. We also provide [a high-level comparison of the two search companies and their products](/learn/what_is_meilisearch/comparison_to_alternatives.md#meilisearch-vs-algolia) elsewhere.
+
+## Migration script
+
+Migrating indexes from Algolia to Meilisearch is a straightforward process:
+
+1. Export your data stored in Algolia
```

Comment:
Can we use each of these steps as an H2 and use the H3s as subheadings for this?

Reply:
Great idea!